k8s/dust-kube-private/network-policies/allow-same-namespace.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
```

Comment:
Can't we just have a single policy ?

Reply:
Nope, theses are temporary (and actually not applied). I forgot to mention but in a subsequent PR, i'll update the policies of private to be extremely restrictive.

Reply:
WDYM not applied ? I am not sure I understand what this does then ?

Reply:
```
# kubectl apply -f "$(dirname "$0")/network-policies/allow-same-namespace.yaml"
# kubectl apply -f "$(dirname "$0")/network-policies/default-deny-ingress.yaml"
```

They are commented out in apply_infra_dust-kube-private as I need to tweak them a bit more.
Do you want me to remove them ? (i kept as ref for my next pr)

Reply:
No of course, feel free to leave them I was just wondering 👍